eduNEXT/eox-audit-model
# perf: add compatibility with palm release

## Description

This PR adds support for the Open edx Palm release.

## Testing instructions

1. Create an environment with Palm release, you can use [Tutor](https://docs.tutor.overhang.io/index.html) or [TVM](https://github.com/eduNEXT/tvm)
2. Install the Django private plugin eox-audit-model using this PR branch, you can use steps in [Tutor documentation](https://docs.tutor.overhang.io/configuration.html#installing-extra-xblocks-and-requirements).
3. Configure [eox-core](https://github.com/eduNEXT/eox-core) as is shown in the [eox-core test cases document](https://docs.google.com/document/d/1gaQyHBGDDkvy_NsFw07MRV6uI2R49CeWCDHzTVi_TiI/edit#heading=h.2tnvhqtb6so6)
4. Run the [test cases](https://docs.google.com/document/d/1v6mxFSf1k7JrxU36Ok9uVDFBhtxVuGmhFLtxtbIsPFs/edit); **in this same file you can find more information about the plugin configurations** 

## Additional information
[JIRA ISSUE DS-707](https://edunext.atlassian.net/browse/DS-707)
